extensions/amp-analytics/0.1/requests.js

```diff
@@ -501,3 +530,54 @@ function expandExtraUrlParams(
 
   return Promise.all(requestPromises).then(() => params);
 }
+
+/**
+ * TODO(#29618): Remove after ampim investigation
+ * @param {string} url
+ * @param {*} info
+ */
+function reportErrorTemp(url, info) {
+  if (!isObject(info)) {
+    return;
+  }
+  const location = parseUrlDeprecated(url);
+  if (location.hostname != _GOOGLE_ACTIVEVIEW_HOSTNAME) {
+    return;
+  }
+  const queryString = parseQueryString(location.search);
+  const requestId = queryString['id'];
+  if (requestId != _GOOGLE_ACTIVEVIEW_REQUEST_ID) {
+    return;
+  }
+  const elementSize = queryString['d'] && queryString['d'].split(',');
+  const viewportSize = queryString['bs'] && queryString['bs'].split(',');
+  if (isArray(elementSize)) {
+    const elementWidth = Number(elementSize[0]);
+    const elementHeight = Number(elementSize[1]);
+    if (elementWidth == 0 || elementHeight == 0) {
+      dev().error(
```

Comment:
ping @lannka and @rcebulko on the format of the error message.

Reply:
Is this an error in something running/loading/unknown source, an error in the ad config, or an error in the publisher page? I ask in case this should be a user error or expected error.
(
'zero element size' is clear in context, but hard to identify by itself without searching for the error in the source code. Slightly more verbose, something like 'Failed to render ad with height or width of zero' will make it easier to parse for the release engineer who sees this error crop up :)

Reply:
😄 I don't know why we are getting this error. In theory the element size should never be zero in the visible request (otherwise it is not visible). We're still trying to investigate potential causes. I wanted to collect it as dev().error to collect at a higher sampling rate. 

>something like 'Failed to render ad with height or width of zero' will make it easier to parse for the release engineer who sees this error crop up :)

It's a good point. I'll add a note to release engineer instructing them to safely ignore the error.

Reply:
SGTM as long as the error message is updated

Reply:
Switched to `Debugging: Activeview request with zero...`